openslides_backend/action/actions/user/create_update_permissions_mixin.py

```diff
@@ -343,10 +343,25 @@ def check_group_D(
                     }
                 )
 
-    def check_group_E(self, permstore: PermissionVarStore, fields: List[str]) -> None:
-        """Check Group E organization_management_level: OML level necessary"""
+    def check_group_E(
+        self, permstore: PermissionVarStore, fields: List[str], instance: Dict[str, Any]
+    ) -> None:
+        """Check Group E organization_management_level: OML level necessary.
+        Also checks if user has higher oml."""
         if fields and permstore.user_oml < OrganizationManagementLevel.CAN_MANAGE_USERS:
             raise MissingPermission(OrganizationManagementLevel.CAN_MANAGE_USERS)
+        user = self.datastore.get(
+            FullQualifiedId(Collection("user"), instance["id"]),
+            ["organization_management_level"],
+        )
+        if (
+            fields
```

Comment:
The restriction should not only apply for changing the OML of another user, but for editing him altogether (that's what I tried to say with https://github.com/OpenSlides/openslides-backend/issues/883#issuecomment-890905647). So if I have OML can_manage_users and X has OML can_manage_organization, then I cannot call `user.update` for him, indipendent on the action data.

Reply:
So it should be a more general check? Okay.

Reply:
Worked on this.
The permission to change a user with higher oml is fully restricted.

Reply:
I also fixed the test problems.